### DISCUSSION TITLE
Success report on Audi A3 8V 2013, installed new firmware/ M.I.B/ new map on a MIB1 unit

### DISCUSSION THREAD

**Discussion by Xendo69** [Mr-MIBonk/M.I.B._More-Incredible-Bash#536] [Show and tell]
Hi
first, I will thanks all people that implemented the version of M.I.B 3.7.1 MHIG edition, it worked fine on my unit.
What I had at the beginning : Audi A3 2013 with MIB1 with firmware version MHIG_EU_AU_S0337.
1. I activated and tested the "greenMenu" with OBDEleven-> abo 1 year(need to buy also the stick :-/ )
2. I installed the last firmware version MHIG_EU_AU_K1555. Tested if all ok (sound , map etc)
3. I installed and ran the M.I.B patch. Before I did a backup (menu in the M.I.B)
4. When done, I controllled that I have the good key and valid
5. Now i'm installing the last version of the map, with the normal menu.

best greetings

**Comment by Misterious30** [comment]
Hi Xendo69,

Can you tell me which file from the patches from the patch folder located in https://mibsolution.one/ M.I.B did you use?  what activation key did you have? I cannot see your version  MHIG_EU_AU_S0337 in that folder either. Did you copy all patches over to the SD and patch the head unit? 

Thanks

**Reply by MGZTT190** [reply]
Hello  i am just trying to build a SD card to do the updates,  the mibsolution.one web site seems to be unavailable now  any one know if this is temporary or if this is now not accessible any more

Cheers

d

**Reply by Xendo69** [reply]
Try here : https://github.com/olli991/mib-std2-pq-zr-toolbox or here https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/releases

**Reply by MGZTT190** [reply]
Hi Xendo69

Its the patches for the patch folder that we cant get because  https://mibsolution.one/  is no longer working.  I was researching this last week but didn't have any sd cards and logged with guest / guest and saw the files required, but didn't down load anything because of the lack of SD cards now i have them i have tried to go back but the site is down.  is there a mirror or may be someone who has down loaded the patches pack for the MHIG system can make it available somewhere else.

On another point the 2024 maps pack for the MHIG when un zipped wont fit on a 32gb sd card  formatted as FAT32 do you have to have the MIB2 folder on the sd card if i have a MHIG system because i think that's a MIB1 unit.  Else will exFAT32 work for a 64GB card? as these don't seem to want to format as FAT32 no matter which tool you use.

Cheers

david

**Reply by Xendo69** [reply]
If you have the version 3.7.1 the patch is already include. Here is the link of archive 3.7.1, unzip and direct copy on SD card (without folder)
https://limewire.com/d/jWQQS#d2wllGxMc7

For the map, depending of the region, I assume, but mine (europe) is 29,6 Go, so it going on a 32GB SD card.

hope that will help you,

greetings

Xendo

**Reply by MGZTT190** [reply]
Many thanks,  so the 3.7.1 will just install  under the red menu  - do all the firmware bits then I go into the green menu to run the MIB up date all and it will up date all the apps and install the code to allow all map updates,  correct?

then to the maps

for the map file i have down loaded P350_N60S5MIBH3_EU.7z  as linked to from mib-helper.com it un zipps at 29.6 MB  a 32 mb sd card on fat 32 formats to 29.1MB so both mib1 and mib2 folders and metainfo2 file wont fit.  do we need mib2 folder?  or will a 64gb sd card formatted in exFAT32 work?

many thanks for your help on this i dont want to brick my unit

Dave

**Reply by Xendo69** [reply]
Hello

concerning the installation check my first post, if you have the same unit version , you will need maybe to activate the "greenMenu". Check this before to begin something. You will need to access to the green menu to pach your unit.
Then install the last firmware->  then patch unit with 3.7.1 (at the end , if it looks like blocked, dont restart/turn off or take out the sd card, its normal ,wait until the unit reboot 2 or 3 times i don't remember), you will see a message when end. Then check the key under normal menu (sould be something like xxxxxxxxxxxxxFF at the end).
Finally you can install the map.
VERY IMPORTANT : check you battery charge, i connected mine to a charger during all the process. Check if you car have a boose system sound with amplifier or other amplifier, you need to disconnect it before you update your unit, look youtube https://youtu.be/eAQgzYlnSrA)

concerning the MAP, yes you can try on a 64Gb card


hope that help you,

greetings

Xendo

**Reply by MGZTT190** [reply]
hi ya 

ver 3.6.0 of bash says it activates the green menu as it installs,  do you think the ver 3.7.1 will not?  ( i dont have obd11 or any other tools ) i was relying on that fact that 3.6.0 with the patches from mibsolution.one would do that for me.   

 I dont have any extra amps only the std system in a 8v 2014  S3  and will have a charger connected to the battery at all times.

Have you managed to format a 64gb card as FAT32 or do you think a exFAT32 card will work  -  i am guessing that if the unit cannot read a exFAT32 card it just wont do the up date not bugger it all up.

cheers

d

**Reply by MGZTT190** [reply]
I have found a utility on the good old internet that will force format a SD above 32gb as fat32 it gives us about 58gb of space on the sd card but way enough to do the job.   so that's solved,  now just need to understand if the green menu is going to be activated by 3.7.1 or if i need to wait for mibsolutions.one to come back on line and make a 3.6.0 set

cheers

david

**Reply by Xendo69** [reply]
As i remember the version 3.6.0 you need to patch yourself. The version 3.7.1, that I gave you the link, contains all needed correction. Concerning the size of the memory card i dont understand the problem. As soon that you patched the unit (firmware update and patch), you cant broke your unit, it's only map. You can use also the official one from A.u.d.i, you will have the life key installed.....

**Reply by MGZTT190** [reply]
thanks - the sd card size thing was around the format of the card, in windows up to 32gb cards can be fat32 above that are exfat32 they are different formats.  I have found a utility on the internet that has formatted a 64gb card as fat32 so that should work, once i have either 3.7.1 or 3.6.0 with patches working, at the moment i access green menu select the patch updater on the 3.7.1 version of mib and it says there is a check sum error in the patch, so it wont install.

cheers

d

**Reply by Xendo69** [reply]
What firmware did you install ?

**Reply by MGZTT190** [reply]
i dont know it was the one you sent in the link ( If you have the version 3.7.1 the patch is already include. Here is the link of archive 3.7.1, unzip and direct copy on SD card (without folder)
https://limewire.com/d/jWQQS#d2wllGxMc7
i assume this is the last version available for the MHIGH

D

**Reply by MGZTT190** [reply]
i dont suppose anyone knows a contact at mibsolutions.one so we can try to get that back up again so i could try to create a 3.6.0 pack age

d

**Reply by Xendo69** [reply]
The link that I posted isn’t the firmware for your unit. You need to install the last firmware MHIG_EU_AU_K1555 first. After only, you can patch your unit with the M.I.B 3.7.1

**Reply by MGZTT190** [reply]
this is what i didn't know,  i have MHIG_eu_AU_S1531_1 and need to go to MHIG_EU_AU_K1555 and i think i need 3.6.0 with the patches from mibsolutions.one site  i have the 3.6.0 on a sd card ready to go but when trying to get into mibsolutions.one there is a web error and the site is not available so i guess what i need is a mirror of the site or copy of what was on mibnsolutions.one or a contact of the webmaster who looks after the site to see if they can get it back up again.

thanks for the help

d

**Reply by vaquero** [reply]
> The link that I posted isn’t the firmware for your unit. You need to install the last firmware MHIG_EU_AU_K1555 first. After only, you can patch your unit with the M.I.B 3.7.1

After re-reading the post I see that Xendo69 indicates that I must first update the firmware and then install and apply M.I.B 3.7.1, because M.I.B 3.7.1 already has the 21000ee patch, but the problem is that I am not able to find the firmware for my car (MHIG_EU_AU_K1555).
Could someone provide me with a copy of it?

**Reply by Xendo69** [reply]
> > The link that I posted isn’t the firmware for your unit. You need to install the last firmware MHIG_EU_AU_K1555 first. After only, you can patch your unit with the M.I.B 3.7.1
> 
> After re-reading the post I see that Xendo69 indicates that I must first update the firmware and then install and apply M.I.B 3.7.1, because M.I.B 3.7.1 already has the 21000ee patch, but the problem is that I am not able to find the firmware for my car (MHIG_EU_AU_K1555). Could someone provide me with a copy of it?

Hello Vaquero
below the link: 
https://limewire.com/d/rOVV7#i3CBWO7Euo
greetings

**Reply by MGZTT190** [reply]
hi ya many thanks this is what i have also been after for the last few weeks  i will now give it a go and see if mine will update.  cheers

david

**Reply by vaquero** [reply]
Thank you very much Xendo69.
I will try, as MGZTT190 also comments, and see how it goes.
Thanks for your help and your quickness

**Comment by Xendo69** [comment]
Hello Misterious

to determine which firmware you need to install go under https://mib-helper.com/ and fill with the firmware that you have currently in your car.

then if it's the same that you need to install (MHIG_EU_AU_K1555):

under https://mibsolution.one/ i used the firmware MHIG_EU_AU_K1555. Look under "MQB_solution->_PRE MIB2" directory, search K1555 and you will find the file. Don't ask me why is the file there.......
Then you must install the key, but before you need to control if you have acces to the green menu. If yes then you can run the MMI_M.I.B._More-Incredible-Bash-3.7.1 with the green menu (google it, you will find a video on that). Then check if the key is in, and then install the last map.

**Reply by Misterious30** [reply]
Hey @Xendo69 

Thanks for the quick response.  Since my last message, I have:

- upgraded firmware to MHIG_EU_AU_K1555
- installed M.I.B into the car using the latest tag from this site 3.7.1

**Note**: before I ran PATCH NOW in 'green menu'....I did run the create backup process (green menu) and it reached a line saying 'backing up backup' and it stayed here about 40mins so I hit back....did the backup you mentioned tell you something like...'you can go back now finished backup' ? 
 
- I ran the PATCH script in the green menu(photo) and it takes me to a blank screen but nothing happens)
_the notes in this release says you do not need a patch file as it will patch the '021000ee' fec code into the car without the need for 20230926_M.I.B_Patches file located at  https://mibsolution.one/ under M.I.B folder_

Am I doing something wrong in the process? 

Thanks for your advice and help 

![PXL_20240920_210602926 MP](https://github.com/user-attachments/assets/45b3604c-17d2-42c9-ba03-c090295a71ea)
![PXL_20240920_210435450 MP](https://github.com/user-attachments/assets/8bc707f8-b693-4780-9080-42d2df8f7100)

**Reply by Misterious30** [reply]
Hey @Xendo69 

I have tried again this morning to patch the FEC code and it has worked.

I tried the backup again before starting the patch.  It completed the backup and told me to go back and then I patched successfully.

The issue I believe I has was the backup was still running and it will not run another script whilst the first script was running. 

Now I need to download the latest maps and I will update it from the main menu like mr-fix from the Youtube video. 

If you have any tips for the map update, let me know?

Many thanks for your response again 👍

**Reply by Xendo69** [reply]
Hello Misterious

for the map you have the direct link on https://mib-helper.com/ , after that you gave the version of your firmware, check below…. Point 3 gps navigation-> different regions 
To install the map , just insert the memory card and select update map in the mmi menu (normal menu). It takes time. But you can do it several times... each time the vehicle is started, the update resumes as long as the memory card is still inserted.

**Reply by AudiTinker** [reply]
Glad you got it sorted @Misterious30. Xando69 is right about the maps. Are you in AU? I couldn't install the P353 ROW maps but P333 one worked. Took about 50 mins

**Reply by Fallemo** [reply]
Hey guys, can you help me?
I upgraded the firmware to the MHIG_EU_AU_K1555 version.
I created the SD with the latest version of M.I.B, installed from the Red Engineering Menu, all was ok, but i don't know what to do next. How do i access the Green Engineering Menu? 

I have OBD, i activated developer mode, but nothing happened. Thank you!

**Reply by Xendo69** [reply]
@Fallemo , you must check according your car model. Activate developer mode isn't enought.....
check this : 
https://www.youtube.com/watch?v=hjS7V6ca6x0
I can confirm that work with this app (activate green menu), i did . but you need to buy the stick and do an abo (70 euros) to activate this feature....... do spend your money to buy point, option will be not available..... only with abo 1 year.....
Still less what A.u.d.i is asking for update map, more than 250 euro.....

**Reply by vaquero** [reply]
Hi.
I have the MHIG_EU_AU_AU_K1549 firmware version on my car, but when I try to download the MHIG_EU_AU_AU_K1555 version cloudfare returns the error: Cloudflare Tunnel error.
I have unlocked the green menu and installed MMI_M.I.B._More-Incredible-Bash-3.7.1, but when I install the program, the key does not appear (21000ee), the only one that appears is the old one (2100018).
I try to look for the patch files for MMI_M.I.B._More-Incredible-Bash-3.7.1 (20230412_m.i.b_patches.7z), but they are not available either. 
Could someone help me?.

Thanks

**Reply by MGZTT190** [reply]
Hi Ya  I am in the same position i have all the bas files and maps ready to go.  however because the mifsolutions.one site is down i cant get the firmware to add to the patch folder.  Has anyone got MHIG_EU_AU_K1555.  

Cheers

d

**Reply by Misterious30** [reply]
Hi check the site again I believe mib solution is back up and sorted. It was a mess a few months ago! Good luck with the process hope you get it sorted!

**Comment by MGZTT190** [comment]
Just tried it but no luck,  i un 7ziped the file that was down loaded  ( MHIG_EU_AU_K1555 MU0415 8V0906961AD.7z )  and copied the contents into to 3.6.0 MIB sd card patches folder then ran the red menu up date again, after which i could open the green menu and run the mib patch all.  the following screen shots seem to indicate it didn't find the patches and didn't update anything because it either couldn't find the up date or it was not on the sd card  it keeps asking to up the FW  all through the error screens  i have MHIG_EU_AU_S1531_1 on my unit and 2015 maps etc as it was from the factory.   the MIB site says i need the patches for MHIG_EU_AU_AU_K1555 which is what i had thought we had created but may be not.  was the 7.zip file for the patches folder in the 3.6.0 version of the updater? 

![error 1](https://github.com/user-attachments/assets/92614aff-cc60-41ef-957a-6fc67bf1ebdc)
![error 2](https://github.com/user-attachments/assets/0eb14e2f-f470-4ad6-9c8a-2b7b83484465)
![error 3](https://github.com/user-attachments/assets/a3ab4395-d3b8-42d7-9d56-db509062fec1)
![error 4](https://github.com/user-attachments/assets/69c75184-237e-4f29-925b-4b0c066187d1)
![error 5](https://github.com/user-attachments/assets/d9f8771f-f469-4454-9c48-565291cc423c)
![error 6](https://github.com/user-attachments/assets/c4cb2acf-4021-4012-ae73-ac120eae3434)


cheers 

d

**Reply by vaquero** [reply]
I think you are messing with the process:

1- https://www.youtube.com/watch?v=SeQCTiXJHVc
2- https://www.youtube.com/watch?v=oQaE-uaMRZY
3- https://www.youtube.com/watch?v=rzOmVFBbDNg
4- https://www.youtube.com/watch?v=e3sMBUWPWZk
5- http://map.audi-mib.ru (most current maps).

Note. check if any step is repeated.

**Reply by vaquero** [reply]
I just updated the firmware of my Audi A3, with Xendo69's file, following the steps in this video:
https://www.youtube.com/watch?v=SeQCTiXJHVc 
and it worked perfectly!

The only problem is that again I have disabled the green menu and I have to re-enable it with the ODB11 to continue the process.

**Reply by MGZTT190** [reply]
Hi

I had mis labelled the SD cards.........   i was trying to load the mib 3.6.0 before the fw update  durrrrrr.    i have the fw updated to the K1555 now but during this update the green menu access has been removed, so i know i need to run the mib install again but now i have the k1555 should i just use the 3.7.1 version that Xendo69 sent over a week or so ago or build a 3.6.0 and see if anyone has the patch folder that we would normally get from mibsolutions.one?   I think from what Xendo69 said now we have K1555 on the unit the 3.7.1 should work ok.  what do we think give it a go and see.

I dont have obd11 or anything like that once you load the 3.6.0 or 3.7.1 it activates the green menu for you.

cheers

d

**Reply by MGZTT190** [reply]
well i just tried the 3.7.1 and it worked, enabled the green menu the patcher updated the codes and the 2024 map up date is running as long as this finished ok that's it all done,  many many thanks to you all.

kind regards

david

**Reply by vaquero** [reply]
I am very pleased.
I do not know what I have not done well, I have tried the process 2 times and always gives me error:
1- I have updated the firmware to version K1555.
2- I have the green menu disabled.
3- I install MIB 3.7.1 and the green menu is enabled.
4- I run the patch in the green menu and I get the following error.

MMX-flash---->
[/net/mmx/fs/sda0/apps/sbin/flashlock] Info : Root-Block at Offset (49)!
ifs-root-stage2.ifs offset: 0x00ba0000
ifs-root-stage2.ifs size: 845DCC00
Found JTB-07422.05.15999614C9 backup, will check if all complete...
Doing standard backup
It will take about 5 minutes to complete.
Please wait...
Version info..... -> Backup is already there!
FEC folders...... -> Backup is already there!
Shadow files..... -> Backup is already there!
Partition info... -> Backup is already there!
Folder lists..... -> Backup is already there!
Datasets......... -> Backup is already there!
EEPROM........... -> Backup is already there!
storage1......... -> Backup is already there!
storage2......... -> Backup is already there!
SWDL............. -> Backup is already there!
GEM jars......... -> Backup is already there!
lsd.jxe.......... -> Backup is already there!
HMI folders...... -> Backup is already there!
RCC OFFSET's..... -> Backup is already there!
RCC Flashlock.... -> Backup is already there!
RCC flash dump... -> Backup is already there!
RCC ifs-stage2... -> Backup is already there!
MMX flash dump... -> Backup is already there!
backing up backup -> Backup part 1 is already there! -> Backup part 2 is already there!

ls -als /net/mmx/mnt/boardbook --> total 256
      64 drwxrwxrwx  4 root      root          32768 Jan 01 00:02 .
      64 drwxrwxr-x  4 root      root          32768 Jan 01 00:03 ..
      64 drwxrwxrwx  3 root      root          32768 Jan 01 00:05 MU0404
      64 drwxrwxrwx  3 root      root          32768 Jan 01 00:05 MU0415
Free space in /net/mmx/mnt/boardbook 756Mbytes

MMX-flash-done!
--- Reading patch DB...
--- Searching offset and size of root-stage2.ifs...:
[/net/mmx/fs/sda0/apps/sbin/flashlock] Info : Root-Block at Offset (49)!
ifs-root-stage2.ifs offset: 0x00ba0000
ifs-root-stage2.ifs size: 845DCC00
--- OK: OFFSETPART2 in DB and RCC flash match.
--- OK: STAGE2SIZE in DB and RCC flash match.
--- Dumping root-stage2.ifs from RCC flash.
    It will take about 1 minute. Please wait...
--- Patching dumped root-stage2.ifs.
    It will take about 2 minutes. Please wait...
!!! Patching failed.
    Stopping here ...
--------------- DEBUG --------------
IFS_VERSION:		9611
OFFSETPART2:		ba0000
OFFSETPART2_DB:		ba0000
STAGE2SIZE:		845DCC00
STAGE2SIZE_orig:	845DCC00
SHA1_orig:		86dab010861fc0f59bb7ca7aaecf1188b30a4142
header_chk_FEC_CP:	905DCC00
SHA1_FEC_CP:		8ad176923b58c3565ad02df0e0640d3745b452d8
IFS2_new:		/net/mmx/fs/sda0/patches/9611-ifs-root-stage2-0x00ba0000-905DCC00_FEC_CP.ifs
IFS2_backup:		/net/mmx/fs/sda0/backup/MU0415-MHIG_EU_AU_K1555-JTB-07422.05.15999614C9/MU0415-ifs-root-part2-0x00ba0000.ifs
Big Endian:		00CC5D84
SHA1_UNIT:		
SHA1_PATCH:		
------------------------------------

timestamp: 1970_01_01_00_05_52----->

MMX-reboot---->
Reboot Unit...
Unit is rebooting now...

Buttons will stop working now
Screen might turn off
Reboot will take ~60s

![IMG_0679](https://github.com/user-attachments/assets/42231c9d-eaee-4d54-a4e6-6c1d849ebdbe)
![IMG_0680](https://github.com/user-attachments/assets/621e1803-9ee3-402f-b733-996f3ba382fa)
![IMG_0681](https://github.com/user-attachments/assets/1a96229d-5f9d-4c7e-8bd4-634fb7177865)

**Reply by MGZTT190** [reply]
you appear to have done exactly the same as i have.  did the firmware up date go ok,  you were going from a different version of the firmware to the K1555 version i wonder if that was the issue?  once i did mine in the correct order it just worked. did you get the list of systems with the OK next to them all when the red menu update finished?   when i ran the 3.7.1 after completing the install of the FW update from the red menu this indicated that only the launcher component was going to be updated and it also worked ok,  did you get the all green success message? after this i opened the green menu and started the patch? and it also finished ok and re booted.   I added nothing to the 3.7.1 down load that Xndo69 sent over.  I am not an expert in this so i am not sure what to do next. sorry

David

**Reply by vaquero** [reply]
Yes, I previously had firmware K1549 and upgraded to K1555.
The whole process was correct, everything ok both in the firmware change and the MIB installation.

The only thing I can think of is that I installed MIB and applied the patch the first time before updating the firmware, but both before and now it did not install the map keys.

I don't know what else to do. I will try a different SD card and this time I will format it on a windows and not a Mac which is what I have done now.

**Reply by MGZTT190** [reply]
it has to be a pc for formatting and constructing the sd cards.  also for the large sd cards ie 64mb for the maps you must use a 3rd party formatting tool as windows wont let you format as 64mb in fat32  it forces you to use exfat32 by default.  i was told this wont work and it must be fat 32.  I did it the wrong way round first.  i had to remake the sd card with the FW up date that he sent over a few days ago run that from the red menu then verify the FW was up dated in the system menu, after al that empty the sd card again and copy the 3.7.1 stuff that was sent a week or so ago and run that from the red menu again.  This will then enable the green menu and that can be opened and the all in one patch run which for me installed the map keys.  the last step was to go into the normal MMI  setup menu and select map update and the 2024 maps installed ok  it took ages may be 2h but it did it.

cheers

d

**Reply by vaquero** [reply]
Well, I don't know what I'm doing wrong, because following your steps I can't install the maps.
I have formatted an SD card in FAT32 format. I managed to install the MIB 3.7.1 patch with this card format.
![IMG_0682](https://github.com/user-attachments/assets/defea73f-4a15-4224-a0d3-2e5c4c78178c)
![IMG_0683](https://github.com/user-attachments/assets/a338d446-cc01-490d-942d-cfbf06c2493d)
![IMG_0684](https://github.com/user-attachments/assets/ae44d97e-a928-4356-99b2-88e013deee35)

What I don't know if the problem is that although the perpetual license appears to me, the license data seems incomplete.
These are the data shown when selecting the original license:
![IMG_0689](https://github.com/user-attachments/assets/507163a2-4b4e-4780-ada9-0a751a1b4357)

And these are the data that it shows me if I select on the perpetual license.
![IMG_0690](https://github.com/user-attachments/assets/24a46eee-6847-4dcc-ba38-4c92d8d6d1ce)

I don't know if because of this above, although I have an empty SD card formatted in FAT32 and with the Europe 2024 map data for MIB1 High [2024 Europe EU: full 26.20 GB] and I have the 2 folders (Mib1 and Mib2) and the file metainfo2.txt in the root of the SD, my car does not allow me to update the map data.
![IMG_0685](https://github.com/user-attachments/assets/4f910a67-7c91-45fb-8c9e-53a33cccf5ad)
![IMG_0687](https://github.com/user-attachments/assets/a6f32f10-66f3-482c-8155-dc4e7c5f74f7)
![IMG_0688](https://github.com/user-attachments/assets/0fb77cad-633d-4166-b7e7-724406e638a5)

So I'm afraid I'm going to be left without updating the maps in my car.

**Reply by MGZTT190** [reply]
I just checked the licences on my car and they look like

![IMG_3630](https://github.com/user-attachments/assets/1c646afc-092b-40e9-bcfe-ea60b36d4b9f)
![IMG_3632](https://github.com/user-attachments/assets/8f2d734f-13a5-4b72-bbc0-8154b9060645)
![IMG_3634](https://github.com/user-attachments/assets/a94a0c4a-d1f8-46d8-aebb-bb0177a80ab7)


so i am not sure why your map would not load

**Reply by MGZTT190** [reply]
is it the correct map version to match the fw  ie rest of world, or europe etc etc

**Reply by vaquero** [reply]
Unbelievable, but true !!!.
Following the instructions on this website:
[http://map.audi-mib.ru ](http://map.audi-mib.ru )
I have tried with a new SD card that I just bought and with this card the map update worked.
![IMG_0696](https://github.com/user-attachments/assets/089d8e88-82d2-4d99-8ba7-9cacbcefc3e6)
![IMG_0702](https://github.com/user-attachments/assets/5d4294e0-3912-4c20-a757-fe06d559bb12)

Thank you for all the help received :)

**Reply by MGZTT190** [reply]
brill i am glad you managed to get it sorted.  looks like we both got there in the end  :-)

**Comment by Kemper197** [comment]
I have also MHIG and latest orginal fw K1555, but then little confused what is needed to complete this ?
- M.I.B
- Latest MAP package (2024 EU)
- patched firmaware ? (this is what i cant find anywhere)

**Comment by edzya** [comment]
Maybe someone can share link to MHIG_EU_AU_K1555 as it is not available on mibsolutions anymore ?

**Reply by Xendo69** [reply]
hello,

yes it's https://mib-helper.com/index.php?train=MHIG_EU_AU_K1555
https://mibsolution.one/index.php/f/234530


greetings

**Reply by edzya** [reply]
> hello,
> 
> 
> 
> yes it's https://mib-helper.com/index.php?train=MHIG_EU_AU_K1555
> 
> https://mibsolution.one/index.php/f/234530
> 
> 
> 
> 
> 
> greetings

Sorry, but mib-helper redirects to mibsolution and once i open that mibsolution link I get error message the file was not found.

**Reply by Xendo69** [reply]
Connect to the first link, then Download. You must connect to a ftp. You will find the file.

**Reply by barreschristophe69** [reply]
Hello, I have the same problem, I tried with FileZilla and the connection to the server is impossible.
Is it possible to find this folder elsewhere?
Thank you in advance

**Comment by phantom83** [comment]
Sorry, but I can't continue.
I updated the firmware to version MHIG-EU-AU-K1555. From the red menu, I update SDcard MIB 3.6.0. Then, from the green menu, I start MIB, but it gets stuck at the message "I can go back." It basically can't find the patch for k1555_mu0415. How do I fix this? Could someone send me the MHIG_EU_AU_K1555 MU 0415 patch and tell me how to add the 21000ee key to update the maps? Thanks.

**Comment by barreschristophe69** [comment]
Hello Indeed the MHIG_EU_AU_K1555 folders are available, but they are not downloadable in the M.I.B_Backup folder while in the MQB_solution/Firmware/Audi folder it's ok

 

 

De : Misterious30 ***@***.***> 
Envoyé : mercredi 27 août 2025 09:39
À : Mr-MIBonk/M.I.B._More-Incredible-Bash ***@***.***>
Cc : barreschristophe69 ***@***.***>; Comment ***@***.***>
Objet : Re: [Mr-MIBonk/M.I.B._More-Incredible-Bash] Success report on Audi A3 8V 2013, installed new firmware/ M.I.B/ new map on a MIB1 unit (Discussion #536)

 

Hi check the site again I believe mib solution is back up and sorted. It was a mess a few months ago! Good luck with the process hope you get it sorted!

**Comment by nakonajo** [comment]
Hello guys, I have an Audi a3 2015, its a korean version and I want to convert to Eu region and use eu maps. 
![activation](https://github.com/user-attachments/assets/25c7899d-e015-4e24-bc18-766659d7f4c0)
![audia3](https://github.com/user-attachments/assets/adb4a29d-2c50-4200-acf7-5e7d06660f50)
![train version](https://github.com/user-attachments/assets/3791342d-9109-42bf-9357-f4c082891e53)
I can access green menu and have mib installed. Can you guys help me out to successfully do that?
